TaskModules/powershell/TaskModuleSqlUtility/SqlUtility.ps1

```diff
@@ -0,0 +1,36 @@
+function ConvertFrom-Json20([object] $InputObject){ 
+    add-type -assembly system.web.extensions
+    $scriptSerializer=new-object system.web.script.serialization.javascriptSerializer
+    #The comma operator is the array construction operator in PowerShell
```

Comment:
Is there any minimum PowerShell version from where the support for array construction using comma operator is?